api/clients/v2/test/relay_payload_retriever_test.go

```diff
@@ -20,22 +20,24 @@ import (
 	testrandom "github.com/Layr-Labs/eigenda/common/testutils/random"
 	contractEigenDACertVerifier "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDACertVerifier"
 	core "github.com/Layr-Labs/eigenda/core/v2"
+	"github.com/Layr-Labs/eigenda/encoding"
 	"github.com/Layr-Labs/eigenda/encoding/kzg"
 	prover2 "github.com/Layr-Labs/eigenda/encoding/kzg/prover"
+	"github.com/Layr-Labs/eigenda/encoding/utils/codec"
 	"github.com/consensys/gnark-crypto/ecc/bn254"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 )
 
 const g1Path = "../../../../inabox/resources/kzg/g1.point"
-const payloadLength = 100
+const maxPayloadBytes = 1025
```

Comment:
Why 1025? Feel like any surprising number should be documented. Aka if its not a power of 2, I feel like a reason should be given, because I naturally wonder: "is the +1 meaningful here? what is it used for? if I want to change it what will break? etc"

Reply:
Did you notice that this is a test file? IMO an unusual upper bound on a test like this has implied documentation, i.e. "I want to test approximately this range, but I want to avoid implying that it must be a power of 2". It's changed in this PR, since I noticed I had used a hardcoded length instead of randomizing, and since I was already making changes, it seemed like 100 bytes was low for the tests.

If we need a comment for `1025`, wouldn't we also need a comment for `1024`? Since the same logic applies, that a reader won't know specifically why `1024` was chosen, and if that specific value is relevant. They might even wrongly assume that a power of 2 is necessary, and be misled.

I guess my question would be, are you saying you think every chosen test value needs a comment?